PyRoute/Galaxy.py

```diff
@@ -195,15 +198,45 @@ def set_bounding_subsectors(self):
 
 class Sector(AreaItem):
     def __init__(self, name, position):
+        name = name.rstrip()
+        if 2 > len(name):
+            raise ValueError("Name string too short")
+        if 2 > len(position):
```

Comment:
The position string should be at least 5 characters long (`# 0,0`)

The name string should be at least three characters long (`# A`) 

If you are going for extreme validation, both position and name (pre-processing) lines should start with `#`.

Reply:
Thanks for that. I'll bolt in your changes and fix the blast damage.

Reply:
Bolted in and pushed out - that's cooking as of writing this comment